packages/main/test/specs/StepInput.spec.js

```diff
@@ -468,6 +468,20 @@ describe("'change' event firing", () => {
 		assert.strictEqual(await input.getProperty("valueState"), valueState, "value state is not changed");
 	});
 
+	it("Value is not rounding when valuePrecision is set, and Value State is set to 'Error' when the value is not compliant", async () => {
+		await browser.url(`test/pages/StepInput.html`);
```

Comment:
Opening a new page without specific need is unnecessary + slowing down the test execution/ I see it's done like this in the whole spec file, so the whole test spec is for refactoring, but it's good start from somewhere.
Ideally we have to open the test page once with all the instances that you want to test. To have clear state you can just use another step input on the page. Only in rare cases you really will need to reopen the test page.